crates/cli-support/src/js/mod.rs

```diff
@@ -1169,10 +1187,10 @@ __wbg_set_wasm(wasm);"
 
         self.write_class_field_types(class, &mut ts_dst);
 
-        dst.push_str("}\n");
+        dst.push('}');
```

Comment:
Lets assume this won't break any output we missed because its not covered by our tests.
:crossed_fingers: